src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritServer.java

```diff
@@ -1066,6 +1093,19 @@ public boolean isGerritSnapshotVersion() {
     }
 
     /**
+     * If Gerrit Missed Events Playback is supported.
+     *
+     * @return true if so, false otherwise.
+     */
+    @JavaScriptMethod
+    public boolean isGerritMissedEventsNotSupported() {
```

Comment:
The javadoc seems to say something different from what the name of the method and its implementation suggests.

And inverse logic like this is usually hard to grok when you read the code, so try to be careful with the naming to not confuse too much.